templates/course/lesson-count.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Lessons count in a Course.
+ *
+ * @package LifterLMS/Templates
+ *
+ * @since [version]
+ * @version [version]
+ *
+ * @author      LifterLMS
+ * @package     LifterLMS/Templates
+ */
```

Comment:
Can you update this docblock considering the one below as template?
https://github.com/gocodebox/lifterlms/pull/2449

Reply:
you mean, like this:

```
/**
 * Template: Lessons count in a Course.
 *
 * @package LifterLMS/Templates
 *
 * @since [version]
 * @version [version]
 */
```

?

Reply:
I gave you a wrong link, not sure why. But yeah something like that.
Although ,the package is
LifterLMS/Templates/Course
see https://github.com/gocodebox/lifterlms/blob/trunk/templates/course/author.php